docs/_docs/ssl-tls.adoc

```diff
@@ -0,0 +1,193 @@
+= SSL/TLS
+
+This page explains how to configure SSL/TLS encryption between the cluster nodes (server and client) and the clients that connect to your cluster.
+
+== Considerations
+
+All internal connections in the cluster context, as well as cluster's user interaction interfaces, are SSL-enabled. The communication categories are as follows: 
+
+* Between the user and the cluster (node): REST
+* Between the user and the platform clients:
+
+** CLI
+** Client
+
+* Between nodes: Network (Messaging, Scalecube)
+
+All SSL configurations activities are performed at the node level.
+
+Due to Micronaut limitations, Apache Ignite does nopt support direct paths to SSL certificates. Instead, it utilizes PKCS12 and JKS keystore.
```

Comment:
Micronaut is an internal implementation of the REST component. I don't think we should highlight this fact in the user documentation.  The Network part is not dependent on micronaut but still utilizes PKCS12 and JKS keystores.